consensus/istanbul/config.go

```diff
@@ -147,17 +149,35 @@ func (c Config) QBFTBlockNumber() int64 {
 
 // IsQBFTConsensusAt checks if qbft consensus is enabled for the block height identified by the given header
 func (c *Config) IsQBFTConsensusAt(blockNumber *big.Int) bool {
-	// If qbftBlock is not defined in genesis qbft consensus is not used
-	if c.TestQBFTBlock == nil {
-		return false
+	if c.TestQBFTBlock != nil {
```

Comment:
should we add a deprecation comment?

Reply:
we added [one](https://github.com/ConsenSys/quorum/blob/master/cmd/utils/flags.go#L2217) for the `config.istanbul` which `TestQBFTBlock` is under it.